src/lbh/components/lbh-tag/_tag.scss

```diff
@@ -0,0 +1,18 @@
+@import "node_modules/govuk-frontend/govuk/components/tag/tag";
+
+@include lbh-exports("lbh/component/tag") {
+  .lbh-tag {
+    @extend %lbh-body-xs;
+    margin-right: govuk-px-to-rem(8);
+    padding: govuk-px-to-rem(2) govuk-px-to-rem(12) 0;
+    color: lbh-colour("lbh-white");
+    background: lbh-colour("lbh-a01");
+    box-shadow: lbh-colour("lbh-primary-inner-shadow") 0 2px 0 0;
+    line-height: govuk-px-to-rem(15);
+    text-transform: uppercase;
+
+    @include govuk-media-query($from: desktop) {
+      margin-right: govuk-px-to-rem(12);
+    }
+  }
```

Comment:
This was lifted directly from `_phase-banner.scss`. There's likely to be some bits of this that aren't relevant in general.

Reply:
@erbridge How would a `tag` be used when it's not in a phase banner? I actually think all of this might be relevant, I just wonder about the margin-right?

Reply:
It's useful for things like task lists: https://design-system.service.gov.uk/patterns/task-list-pages/.

Reply:
In which case @CJWaszczuk please could you move the `margin-right` rules back to the phase banner? Everything else can stay.